test/repro.py

```diff
@@ -0,0 +1,35 @@
+import torch
```

Comment:
What is this file? :)

Reply:
ah lol accidentally checked in my small test file :P sorry